CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 6.1.1
 
+### Fixes
```

Comment:
Should be a new entry under `Unreleased`?

Reply:
Yes, auto-merge failed, thanks.